frontend/src/components/VAudioTrack/VAudioTrack.vue

```diff
@@ -296,9 +296,9 @@ watch(
 
 /* Timekeeping */
 
-const message = computed(() =>
-  activeMediaStore.message
-    ? t(`audioTrack.messages.${activeMediaStore.message}`)
+const message = computed(() => 
+  activeMediaStore.id === props.audio.id && activeMediaStore.message 
+    ? i18n.t(`audioTrack.messages.${activeMediaStore.message}`).toString() 
```

Comment:
@obulat's comment https://github.com/WordPress/openverse/issues/2202#issuecomment-1635406889, although correct in terms of the changes, was a bit out of date. We can access `t` directly as it has been defined in line 70. Also there is no longer a need to use `.toString()`.

```suggestion
    ? t(`audioTrack.messages.${activeMediaStore.message}`)
```

In fact, using `i18n` introduces an error because that's an undefined variable.

```
Error: frontend types: src/components/VAudioTrack/VAudioTrack.vue(301,7): error TS2304: Cannot find name 'i18n'.
```